src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -176,6 +178,13 @@ public static IHostBuilder CreateHostBuilder(IConsole console, string[] urls, st
 
                     services.AddSingleton<IAuthOptions>(authenticationOptions);
 
+                    services.AddSingleton<IEgressOutputOptions>(egressOptions);
+
+                    services.AddMvc(options =>
```

Comment:
Forgot to mention this before, but this should be moved to the `Startup.ConfigureServices` method. We try to keep as much of the HTTP server configuration there as much as possible.

There's an existing call to `AddControllers` there that you can use the overloaded version of it to configure the filters on the MvcOptions instance.